linux/net/rina/rmt.h

```diff
@@ -59,6 +59,7 @@ struct rmt;
 enum flow_state {
 	N1_PORT_STATE_ENABLED,
 	N1_PORT_STATE_DISABLED,
+	N1_PORT_STATE_DO_NOT_DISABLE,
```

Comment:
A variable that represents one of the states of an N-1 port as seen by the RMT (this is all this flow state enum is).. well the best place to put it it's the flow state precisely in my opinion. It is another state of the N-1 port, it means that the N-1 IPCP already re-enabled the port and therefore must not be disabled. And it is implementation specific, there are multiple ways you can implement the interaction between layers.

And in any case, the RMT maintainer should be the first one to judge this. If the maintainer thinks this enum shouldn't be used this way, let's go for the variable approach. No big deal :), but we should stick to the procedures (Maintainers ack/nack their maintained files unless they can't do it for some reason).